src/collections/src/NICollectionViewCellFactory.h

```diff
@@ -166,6 +166,22 @@ _model.delegate = (id)[NICollectionViewCellFactory class];
 
 @end
 
+/**
+ * A protocol that objects may conform to in order to provide a custom suffix to the reuse
+ * identifier of its cell.
+ *
+ * This is useful for objects that can support different configurations, such that cells updated
+ * with an instance of the object can be reused only for objects supporting the same configuration.
+ * This can be used as an optimization for objects that support dynamic configurations that would be
+ * inefficient to reset during -prepareForReuse.
+ */
+@protocol NICollectionViewCellReuseIdentifierExtension <NSObject>
+
+/** A unique reuse identifier suffix to append to the reuse identifier of the cell. */
```

Comment:
Please mention that you can return nil or the empty string to opt out of the suffix.

Reply:
Done. PTAL.